src/lib.rs

```diff
@@ -808,7 +808,13 @@ impl Config {
                 {
                     cmd.env("MAKEFLAGS", makeflags);
                 }
-                _ => {}
+                _ => {
+                    if let Ok(s) = env::var("NUM_JOBS") {
+                        if !cmake_has_parallel(&executable) {
+                            cmd.env("MAKEFLAGS", format!("-j{}", s));
+                        }
```

Comment:
Can this be folded into the processing of `NUM_JOBS` below so the env var and the `cmake --version` command doesn't run twice?